dataeng/jobs/analytics/ProgramEnrollmentReports.groovy

```diff
@@ -21,7 +21,7 @@ class ProgramEnrollmentReports {
                 stringParam('OUTPUT_ROOT', allVars.get('OUTPUT_ROOT'), 'Report output location')
                 stringParam('VERTICA_CREDENTIALS', allVars.get('VERTICA_CREDENTIALS'), 'The path to the Vertica credentials file.')
             }
- 
+            authorization common_authorization(allVars)
```

Comment:
You will need to add the `common_authorization` import above, too, I believe.

Reply:
👍 ahh yup, thanks